### ISSUE TITLE
add screenshot for send2cgeo

### ISSUE THREAD

**Issue by Bananeweizen** [cgeo/cgeo.github.io#87]
The send2cgeo page should contain a screenshot in the description paragraph.

**Comment by Lineflyer** [comment]
What would you expect to see there?
A screenshot of the send2cgeo button on the gc.com page (live map and cache details) or something else?

**Comment by Bananeweizen** [comment]
Yes, just that button with its surroundings, best with some additonal highlighting of the button. Imagine a user that has _never_ used send2cgeo before. How does he/she verify that the script is working? He/she doesn't have any idea what that script is really doing to his browser.

An image is just way more descriptive than the words "adds a button". :)
